content/en/docs/azure/azureEndtoEnd.md

```diff
@@ -117,6 +117,13 @@ To create an app registration:
     alt="Client secret location"
     class="mt-3 mb-3 p-3 border border-info rounded">
 
+### Add "reader" permission for subscription first
+
+1. search for and select **Subscriptions**, or select **Subscriptions** on the **Home** page.
+2. Select **Access control (IAM)** from the sidebar. Select Add a role assignment.
```

Comment:
Suggestion: Split each instruction onto its own line. For example:

2. Select **Access control (IAM)** from the sidebar. 
3. Select Add a role assignment.